tests/testthat/test-unnecessary_placeholder_linter.R

```diff
@@ -1,25 +1,40 @@
-test_that("unnecessary_placeholder_linter skips allowed usages", {
-  # . used in position other than first --> ok
-  expect_lint("x %>% foo(y, .)", NULL, unnecessary_placeholder_linter())
-  # ditto for nested usage
-  expect_lint("x %>% foo(y, bar(.))", NULL, unnecessary_placeholder_linter())
-  # . used twice --> ok
-  expect_lint("x %>% foo(., .)", NULL, unnecessary_placeholder_linter())
-  # . used as a keyword argument --> ok
-  expect_lint("x %>% foo(arg = .)", NULL, unnecessary_placeholder_linter())
-})
+skip_if_not_installed("patrick")
```

Comment:
hmm at this point I think we have enough of our test suite using Patrick that we should make it required & skip this overhead...

Reply:
#1682